src/Aspire.AppHost.Sdk/Aspire.AppHost.Tasks/Aspire.AppHost.Tasks.csproj

```diff
@@ -1,17 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
-    <RollForward>Major</RollForward>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
I chose ns2.0 so that we should work in VS and in CLI. I've tested both and that does seem to be the case.